extensions/wikia/EmbeddableDiscussions/styles/EmbeddableDiscussions.scss

```diff
@@ -12,21 +14,24 @@ $upvote-icon-tiny: '/extensions/wikia/EmbeddableDiscussions/images/upvote-tiny.s
 $share-icon: '/extensions/wikia/EmbeddableDiscussions/images/share.svg';
 $plus-icon: '/extensions/wikia/EmbeddableDiscussions/images/plus-x-large.svg';
 
-$gray: #999;
 $card-border-color: $color-page-border;
 $container-border-color: lighten($color-page-border, 10%);
 
-$avatar-size: 30px;
+// spacing
+$content-spacing-xxsmall: 6px;
+$content-spacing-xsmall: 10px;
+$content-spacing-small: 12px;
+$content-spacing-small-plus: 14px;
+$content-spacing-regular: 18px;
+$content-spacing-large: 24px;
+$content-spacing-xxlarge: 48px;
```

Comment:
Can you link a ticket to tech-debt removal one? It seems we'll have those defined in two places